src/markup/patterns/drizzle/partials/stylesheet-link-tags.hbs

```diff
@@ -0,0 +1,3 @@
+<link rel="stylesheet" href="/css/drizzle.css">
```

Comment:
I really don't like having these `drizzle.css` and `design-system-interface.css` files as separate things. But apparently `drizzle.css` doesn't live in this codebase, so I can't think of a good way to merge these.

The JS has the same problem.